File: video-service/src/main/java/com/service/page/MainPage.java
package com.service.page;

import com.automation.remarks.kirk.Browser;
import com.automation.remarks.kirk.KElement;
import com.automation.remarks.kirk.KElementCollection;
import com.automation.remarks.kirk.Page;

/**
 * Created by sepi on 26.08.17.
 */
public class MainPage extends Page {

  public KElement logo = element("a.navbar-brand");
  public KElement usersTab = element("#users_link");
  public KElementCollection videoFiles = all("[data-parent='#accordion'] strong");


  public MainPage(Browser browser) {
    super(browser);
  }

  public void uploadVideo(String path) {
    element("#filestyle-0").setValue(path);
    element("#upload_submit > button").click();
  }
}

File: src/main/kotlin/pages/MainPage.kt
package pages

import com.automation.remarks.kirk.Page

/**
 * Created by sergey on 17.06.17.
 */
class MainPage : Page() {
    val logo = element("a.navbar-brand")
    val uploadInput = element("#filestyle-0")
    val uploadBtn = element("#upload_submit > button")
}